docs/user/ppl/cmd/addcoltotals.rst

```diff
@@ -0,0 +1,86 @@
+==========
```

Comment:
please update the index file https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/index.rst